plugins/woocommerce/tests/php/src/Internal/Admin/Navigation/MenuReconcilerTest.php

```
@@ -0,0 +1,231 @@
+
+	/**
+	 * The woocommerce_admin_menu_tree filter is applied and receives the
+	 * raw $menu and $submenu.
+	 */
+	public function test_filter_receives_raw_menu_and_submenu() {
+		global $menu, $submenu;
+		$menu    = array( array( 'WooCommerce', 'read', 'woocommerce', '', '' ) );
+		$submenu = array();
+
+		$captured_raw_menu    = null;
+		$captured_raw_submenu = null;
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$captured_raw_submenu'. (undefined)

(UnusedLocalVariable)